include/crpropa/module/PhotoPionProduction.h

```diff
@@ -103,15 +114,42 @@ class PhotoPionProduction: public Module {
 		bool antiNucleons = false,
 		double limit = 0.1,
 		bool haveRedshiftDependence = false);
+
+	// set the target photon field
 	void setPhotonField(ref_ptr<PhotonField> photonField);
+
+	// decide if secondary photons are added to the simulation
 	void setHavePhotons(bool b);
+
+	// decide if secondary neutrinos are added to the simulation
 	void setHaveNeutrinos(bool b);
+
+	// decide if secondary electrons are added to the simulation
 	void setHaveElectrons(bool b);
+
+	// decide if secondary anti nucleons are added to the simulation
 	void setHaveAntiNucleons(bool b);
+
+	// decide if redshift dependent tabulated loss rates are used
 	void setHaveRedshiftDependence(bool b);
+
+	/** Limit the propagation step to a fraction of the mean free path
+	 * @param limit fraction of the mean free path
+	 */	
 	void setLimit(double limit);
+
+	/** set a custom interaction tag to trace back this interaction
+	 * @param tag string that will be added to the candidate and output
+	 */
 	void setInteractionTag(std::string tag);
+
 	void initRate(std::string filename);
+
+	/** get the mean free path for a nucleon
+	 * @param gamma 	Lorentz factor of the nucleon
+	 * @param z 		redshift
+	 * @param onProton 	true for protons, false for neutrons
+	 */
 	double nucleonMFP(double gamma, double z, bool onProton) const;
 	double nucleiModification(int A, int X) const;
```

Comment:
Was that comment intentionally left out?

Reply:
I have added also a comment for the nuclei modification.